src/Projects/JavascriptPrep.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Image } from 'react-bootstrap';
+import * as CONSTANTS from '../constants/constants';
+
+export default function JavascriptPrep(props) {
+    return (
+        <div>
+            <p>
+                Node JS bot that takes in a person's name and phone number along with their district id and keeps checking for vaccination slots' availability every minute.
```

Comment:
why is vaccine finder project's content in javascriptprep file?

Reply:
Content not updated :P